test/common.py

```diff
@@ -1288,10 +1288,7 @@ def clean_js_output(self, output):
     long_lines = []
 
     def cleanup(line):
-      if len(line) > 2048:
-        # Sanity check that this is really the emscripten program/module on
-        # a single line.
-        assert line.startswith('var Module=typeof Module!="undefined"')
+      if len(line) > 2048 and line.startswith('var Module=typeof Module!="undefined"'):
```

Comment:
What new pattern of size > 2048 is now being accepted here, aside from the program/module on a single line as the old comment used to say?

Reply:
We had one test where we have line over 2kb but it was coming from a long line which was the minified wasm2js output, not the overall emscripten output, so it didn't start with `var Module..`

So now, such tests would not get this "REPLACED ENTIRE PROGRAM ON SINGLE LINE" hack.. only when we is clearly the entire emscripten program minified do we do this trick?  We could come up with a pattern to detect wasm2js output but I don't think its worth it.

Reply:
BTW the `len(line) > 2048` condition was always there, I just most the assertion up into a condition becuase it was firing.

Reply:
Yes, I understand that the 2048 is pre-existing, but sorry, I'm still not following what changed here. If the long content is minified wasm2js output, why is it printed out after this PR, but not before?

Reply:
Because one of the consequences of this change is that when node prints the line that threw an error is not prints a more accurate line, rather then printing the location of the rethrow.

Because we are no longer rethrowing at least on error is now (correctly) reported as coming from inside of wasm2js content rather than the minified emscripten-generated JS.

This code here in common.py avoids echoing the entire emscripten-genertaed module, but it doesn't (currently avoid) echoing the asmjs code (which appears on a different line).

Reply:
I see now, thanks.